app/Http/Requests/TwilioRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+use App\PhoneNumber;
+use App\Friend;
+use Illuminate\Foundation\Validation\ValidatesRequests;
+
+class TwilioRequest extends Request
+{
+    /**
+     * Validate that the incoming request is from twilio.
+     *
+     * @return bool
+     */
+    public function authorize(\Services_Twilio_RequestValidator $validator)
+    {
+        if (app()->isLocal()) {
+            return true;
+        }
+
+        return $validator->validate(
+            $this->headers->get('X-Twilio-Signature'),
+            $this->fullUrl(),
+            $this->input()
+        );
+    }
+
+    public function rules()
+    {
+        return [];
+    }
+
+    public function phoneNumber()
+    {
+        $number = PhoneNumber::byNumber($this->input('From'))->first();
```

Comment:
This was previously using "findVerifiedByTwilioNumber".. now it's just using `byNumber`. Are we going to miss them because they need to be translated, or has there been a change to the syntax?